packages/flutter_goldens_client/lib/skia_client.dart

```diff
@@ -67,17 +66,19 @@ class SkiaGoldClient extends GoldensClient {
   /// The [workDirectory] parameter specifies the current directory that golden
   /// tests are executing in, relative to the library of the given test. It is
   /// informed by the basedir of the [FlutterSkiaGoldFileComparator].
-  Future<void> auth() async {
+  Future<void> auth(Directory workDirectory) async {
+    assert(workDirectory != null);
```

Comment:
Maybe add an error message here indicating that auth can only be called once?

Reply:
I don't know that I would want to throw an error if it is already authorized, I just made it a no-op in that case.